README.md

```diff
@@ -195,3 +195,73 @@ We decided to use [Mongo DB](https://www.mongodb.com/) as a database. It is a do
 
 *See more information on why to use Mongo DB [here](https://www.mongodb.com/why-use-mongodb#:~:text=Companies%20and%20development%20teams%20of,of%20both%20data%20and%20traffic.).*
 
+## Milestone :six:
+*Deadline: 07.01.2021*
+
+### Overview
+The task for this milestone was to implement the basics of one tool and to save the collected values of this tool.
+We implemented Code Charts mostly by using Pair-Programming. To get the GUI working we used a framework called [TornadoFX](https://edvin.gitbooks.io/tornadofx-guide/content/)
+
+### StringHandler
+This class is not supposed to provide any GUI elements, but it is able to randomly generate Strings with the needed length to generate enough Strings in a reasonable time. For this functionality we used a equation to calculate this length:
+$$
+\frac{\ln{(input)}} {\ln{(alphabetLength)}}.roundToInt() + 1
```

Comment:
Das sollten wir evtl. noch ändern.

Reply:
Ja, hat in meinem Editor funktioniert, da schaue ich nochmal

Reply:
Ok, sollte nun funktionieren.